recipe/meta.yaml

```diff
@@ -100,6 +100,7 @@ outputs:
     requirements:
       run:
         - {{ pin_subpackage(name + '-grpcio-gcp', exact=True) }}
+        - grpcio-gcp >=0.2.2
```

Comment:
I'm pretty sure this line is unnecessary. pin_subpackage with exact=True means that google-api-core-grpcgcp depends on exactly the same dependencies as google-api-core-grpcio-gcp

https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#referencing-subpackages

```suggestion
```

Reply:
I agree that

```
        - {{ pin_subpackage(name, exact=True) }}
        - grpcio-gcp >=0.2.2
```

would match `setup.py` more closely.

Reply:
Let's consider as a separate PR. I will merge #96 once tests pass.